### PR TITLE
Add option to automatically jump to the next unlabeled data folder

### LINKED ISSUE #1221: Feature request_Mark labeled video's folder
**Is your feature request related to a problem? Please describe.**
Hello, 
I'm afraid of this being taken as just a request of a lazy person...But I'm labeling usually between 50 to 260 videos, sometimes more. Once I label one video, I have to search for videos I didn't label yet. My only indication on which folder Isn't labeled yet is the date of modification so during the frame labeling process, I just sort folders by date of modification. This works well for initial frame labeling.

The problem is during the refining label process after video analysis. The machinelabels-iter0.h5 file is inside a folder named after the video,  after labeling, the folder's date of modification doesn't change so I have to check again inside the folders if the files where modified and this takes a very long time when you have to refine label on hundreds of videos or more...

Notice on the screenshots that the folder modification date doesn't actualize after labeling while the file inside does.

**Note also that this bug only happens during label refinement after videos analysis, the folders modification date is changing during initial frame labeling. This bug is specific to refine_labels.**

which folder did I already labeled
Oh this file Is already labeled

**Describe the solution you'd like**
It would be really cool if the already labeled folder showed a green mark for instance saying you already labeled the file inside this folder. Or make the labeled file temporary masked so we could find easily the files awaiting labeling.
But correcting the bug so that already labeled folder actualize their modification date would be enough.

**Describe alternatives you've considered**
Take a screenshot of folder list then marking manually those I already labeled

**Additional context**

### PR DESCRIPTION
**Pull request by jeylau**
Although the feature will be nicely incorporated in the new GUI (that is, with a nice workspace dock widget), labeling/refinement GUIs can already now be set to automatically open the next unlabeled data folder upon passing `jump_unlabeled=True` to `deeplabcut.label_frames()` or `deeplabcut.refine_labels()`.
This is done by recursively identifying the h5 file most recently modified and selecting the folder coming next alphabetically.

Fixes #1221

### CODE REVIEW

**Review by MMathisLab — APPROVED** [review]

**Review by aristotelis-economides — APPROVED** [review]

**Comment by MMathisLab** [comment]
@jeylau you had marked as draft; so I will wait for you to decide to merge

**Comment by jeylau** [comment]
@MMathisLab, perfect. I'll merge after further testing :)

**Review by AlexEMG — APPROVED** [review]

**Review by MMathisLab — APPROVED** [review]
did not directly test, but looks fine!

**Pull request merged**
